Makefile

```diff
@@ -7,19 +7,22 @@ help:
 	@echo "  man      to build the man file from README.rst"
 	@echo "  install  to install. Use PREFIX and MANPREFIX to customize."
 
+generate_script:
+	@./inject_version.sh
+
```

Comment:
I'm concerned with the complexity added by the unversioned `.build` script. Could we instead just modify a version string within the script?

Reply:
in my world, anything manual is considered evil due to human error, required human work, etc unless one can articulate good reason not to automate something. Complexity is usually ground more in how automation is performed. I don't see complexity in the proposed method, however, I'm open to folks articulating their perspective in articulating complexity.

Reply:
I propose that inject_version.sh would still exist, but instead of copying the unversioned script to the versioned script it would directly modify the version string within the sole copy of the script. You might use `sed` to replace `^VERSION = ".*"$` with `VERSION = "NEW.VERSION.NUMBER"`

Reply:
> I propose that inject_version.sh would still exist, but instead of copying the unversioned script to the versioned script it would directly modify the version string within the sole copy of the script. You might use `sed` to replace `^VERSION = ".*"$` with `VERSION = "NEW.VERSION.NUMBER"`

`sed` would be one way, though not my favorite. I will try `envsubst` first and if that doesn't work then I'll go with `sed`. I'll fix this up.